src/utils/index.js

```diff
@@ -1,8 +1,9 @@
 import aws_config from '../aws-config';
-
-export const API_URL = process.env.REACT_APP_STAGE  === 'production'
-    ? process.env.REACT_APP_PROD_API
-    : process.env.REACT_APP_STAGING_API
+//COMMENT OUT FOR TESTING, USING STAGING NOW
```

Comment:
The code that you commented out will always use the staging api when developing locally

Reply:
Updated